helm/csi-vxflexos/values.yaml

```diff
@@ -196,21 +196,23 @@ vgsnapshotter:
 # Enable this feature only after contact support for additional information
 podmon:
   enabled: false
-  image: dellemc/podmon:v1.1.0
+  image: dellemc/podmon:v1.2.0
   #controller:
   #  args:
   #    - "--csisock=unix:/var/run/csi/csi.sock"
   #    - "--labelvalue=csi-vxflexos"
   #    - "--mode=controller"
   #    - "--skipArrayConnectionValidation=false"
```

Comment:
Can you add "--arrayConnectivityPollRate=5" and "--arrayConnectivityConnectionLossThreshold=3" to align with what we have in the [documentation](https://dell.github.io/csm-docs/docs/resiliency/deployment/#powerflex-specific-recommendations)?

Reply:
Those are optionally can be added to configmap for user to dynamically update, reason they aren't in default values.yaml.